docs/managing_deis/running-stateless-cluster.rst

```diff
@@ -0,0 +1,213 @@
+:title: Running a stateless cluster
+:description: Configuring the cluster to remove Ceph from the control plane.
+
+.. _running-stateless-cluster:
+
+Running a stateless cluster
+===========================
+
+.. include:: ../_includes/_ceph-dependency-description.rst
+
+This guide is intended to assist users who are interested in removing the Ceph
+dependency of the Deis control plane.
+
+.. note::
+
+  This guide was adapted from content graciously provided by Deis community member
+  `Arne-Christian Blystad`_.
+
+Requirements
+------------
+
+External services are required to replace the internal store components:
+
+* S3-compatible blob store (like `Amazon S3`_)
+* PostgreSQL database (like `Amazon RDS`_)
+* Log drain service with syslog log format compatibility (like `Papertrail`_)
+
+Understanding component changes
+-------------------------------
+
+Either directly or indirectly, all components in the :ref:`control-plane`
+require Ceph (:ref:`store`). Some components require changes to accommodate
+the removal of Ceph. The necessary changes are described below.
+
+Logger
```

Comment:
A lot of this information is very useful; we should possibly take some of this info and dump it into the [component docs](http://docs.deis.io/en/latest/understanding_deis/components/) once we merge this :+1: